CODEOWNERS

```diff
@@ -971,8 +971,8 @@ build.json @home-assistant/supervisor
 /tests/components/otbr/ @home-assistant/core
 /homeassistant/components/ourgroceries/ @OnFreund
 /tests/components/ourgroceries/ @OnFreund
-/homeassistant/components/overkiz/ @imicknl @vlebourl @tetienne @nyroDev
-/tests/components/overkiz/ @imicknl @vlebourl @tetienne @nyroDev
+/homeassistant/components/overkiz/ @imicknl @vlebourl @tetienne @nyroDev @tronix117
+/tests/components/overkiz/ @imicknl @vlebourl @tetienne @nyroDev @tronix117
```

Comment:
Would you be willing to make other contributions / help maintain this integration in the future? 

See https://github.com/home-assistant/architecture/blob/master/adr/0008-code-owners.md.

Reply:
I will, I got some IO devices at home (and I guess for a very long time), and I will want to improve their integration.

Reply:
Great! Are you on Discord by any chance? We have a Discord channel with more devs (very inactive at the moment).

Reply:
Same pseudo ;)